base_layer/wallet/src/transaction_service/config.rs

```diff
@@ -28,6 +28,7 @@ const LOG_TARGET: &str = "wallet::transaction_service::config";
 #[derive(Clone)]
 pub struct TransactionServiceConfig {
     pub base_node_monitoring_timeout: Duration,
+    pub chain_monitoring_timeout: Duration,
```

Comment:
If we are separating these two timeouts into distinct config values then you should rename the `base_node_monitoring_timeout` to `broadcast_monitoring_timeout`

Reply:
Saw that relation just now